### ISSUE TITLE
error in each over range

### ISSUE THREAD

**Issue by sdedeo** [watt/ruby_patience_diff#1]
In sequence_matcher.rb, line 149, I think there should be parentheses:

```
    (0...(a_hi-a_mid)).each do |i|
      yield [a_mid+i, b_mid+i]
    end
```

rather than

```
    0...(a_hi-a_mid).each do |i|
      yield [a_mid+i, b_mid+i]
    end
```

which causes ruby to crash with something like "NoMethodError: undefined method `each' for 1:Fixnum"

**Comment by watt** [comment]
I think the problem you're seeing was actually fixed in version 1.1, but the gem was never actually released. I've pushed a new version to rubygems.org, so you can update your installed gem to get the fix.

**Comment by sdedeo** [comment]
Thank you!

Simon DeDeo
Indiana University & the Santa Fe Institute
http://homes.soic.indiana.edu/sdedeo/
